editor/editor_command_palette.cpp

```diff
@@ -256,7 +256,7 @@ void EditorCommandPalette::register_shortcuts_as_command() {
 		ev.instantiate();
 		ev->set_shortcut(shortcut);
 		String shortcut_text = String(shortcut->get_as_text());
-		add_command(command_name, E.key, callable_mp(EditorNode::get_singleton()->get_viewport(), &Viewport::push_unhandled_input), varray(ev, false), shortcut_text);
+		add_command(command_name, E.key, callable_mp(EditorNode::get_singleton()->get_viewport(), &Viewport::push_input), varray(ev, false), shortcut_text);
```

Comment:
This doesn't seem necessary 🤔

Reply:
The other changes of this PR cause, that `push_unhandled_input` no longer propagates events to SubViewports.
So this change becomes necessary in order to allow shortcuts to work also within SubViewports.

Reply:
But this is editor-only code. Are there any SubViewports that are affected?

Reply:
I was not aware that the editor currently doesn't use SubViewports.

Is it possible to create add-ons or editor-extensions, that utilize shortcuts in SubViewports?
If that is the case, then it would be prudent to make the switch from `push_unhandled_input` to `push_input`.

I would consider the `push_input`-variant more robust than `push_unhandled_input`.

Reply:
Well, this change doesn't seem to have any unwanted side-effects, so it's fine to keep it I guess.
> Is it possible to create add-ons or editor-extensions, that utilize shortcuts in SubViewports?

I think in this case not. The shortcut commands of CommandPalette are not exposed to scripting.